c_glib/test/run-test.sh

```diff
@@ -20,7 +20,7 @@
 test_dir="$(cd $(dirname $0); pwd)"
 build_dir="$(cd .; pwd)"
 
-modules="arrow-glib arrow-gpu-glib parquet-glib"
+modules="arrow-glib arrow-gpu-glib parquet-glib gandiva-glib"
```

Comment:
`arrow-glib arrow-gpu-glib gandiva-glib parquet-glib` order is better.

Reply:
I've fixed it.